City-of-Pori/pori-kada
# PORI-374 Changed the section title to smaller size

To test:

1. drush cc all
2. Create a site section if you don't already have one. (/satakunnan-pelastuslaitos might be available if you have synced lately)
3. Make sure the title of the site is written in smaller font size than before. Compare to https://www.pori.fi/satakunnan-pelastuslaitos
